lib/mutant/expression.rb

```diff
@@ -2,83 +2,21 @@ module Mutant
 
   # Abstract base class for match expression
   class Expression
-    include AbstractType, Adamantium::Flat, Concord::Public.new(:match)
+    include AbstractType, Adamantium::Flat
 
-    include Equalizer.new(:syntax)
+    fragment             = /[A-Za-z][A-Za-z\d_]*/.freeze
```

Comment:
This is just an lvar used to make the definition of `SCOPE_NAME_PATTERN` more readable.

I do not need to access the `fragment` outside this scope anywhere, so it does not make sense to assign it to a constant that would be accessible in other scopes.